addons/misra.py

```diff
@@ -2043,7 +2043,7 @@ def reportError(self, location, num1, num2):
             elif len(self.ruleTexts) == 0:
                 errmsg = 'misra violation (use --rule-texts=<file> to get proper output)'
             else:
-                return
+                errmsg = 'misra violation (rule text is missing in %s)' % args.rule_texts
```

Comment:
this is intentional. people don't want to use all misra checks. so I have said that they only need to write the texts for the rules they want to use.

but I think it could be interesting to report missing rule texts somehow in case that is by mistake.. do you have some suggestion? Maybe an "--verify-rule-texts" could output the rule numbers with/without rule text.

Reply:
Got it.

I though they might use rules suppression file to skip undesired checks, but adding `verify-rule-texts` is okay too. The thing is, we might not notice that we fill rules files incorrectly, when we use `pdf2text` tool for example.

I'll add verification routine in next patches, and also something like `--list-text-rules` would be useful.